go-controller/pkg/testing/libovsdb/libovsdb.go

```diff
@@ -149,9 +149,15 @@ func newOVSDBServer(cfg config.OvnAuthConfig, dbModel *model.DBModel, schema ovs
 			if err != nil {
 				return nil, err
 			}
-			res, _ := db.Insert(dbName, tableName, uuid.NewString(), row)
-			if res.Error != "" {
-				return nil, fmt.Errorf("%s: %s", res.Error, res.Details)
+			tableUpdate := make(ovsdb.TableUpdate)
+			tableUpdate.AddRowUpdate(uuid.NewString(), &ovsdb.RowUpdate{
+				New: &row,
+			})
+			tableUpdates := make(ovsdb.TableUpdates)
+			tableUpdates.AddTableUpdate(tableName, tableUpdate)
+			err = db.Commit(dbName, tableUpdates)
+			if err != nil {
+				return nil, err
```

Comment:
This is fixed in https://github.com/ovn-org/ovn-kubernetes/pull/2369 with the added bonus of named uuid support.

Reply:
rebased so this is brought in